### PR TITLE
feat: add background image and frosted glass welcome box

### PR DESCRIPTION
**Pull request by incredible-elk**

### CODE REVIEW

File: styles/Home.module.css [hunk 1]
```diff
@@ -1,47 +1,71 @@
...
 }
+
 .boxContainer {
-  min-height: 100vH;
+  min-height: 100vh;
   display: flex;
   flex-direction: column;
   align-items: center;
   justify-content: center;
+  background: url(../img/flipped-ice-aleksandra-khaprenko.jpeg) no-repeat center
```
Comment (bxt) [inline styles/Home.module.css:11]: It seems the [unsplash license](https://unsplash.com/license) does not require to put an attribution for this image, but I think we should put it in. Let's do that in another PR maybe? Where did you get the image from?

**Review by bxt — APPROVED** [review]

File: styles/Home.module.css (hunk 1 above)
Comment (incredible-elk) [reply styles/Home.module.css:11]: Great idea. That link was provided by the photographer: "Photo by <a href="https://unsplash.com/@xaprrr?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Aleksandra Khaprenko</a> on <a href="https://unsplash.com/?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a>"
  
we can use that later on.
